tools/topology/sof-icl-rt711-rt1308-rt715-hdmi.m4

```diff
@@ -13,6 +13,8 @@ include(`common/tlv.m4')
 # Include Token library
 include(`sof/tokens.m4')
 
+include(`platform/intel/dmic.m4')
```

Comment:
@libinyang are you aware that we don't use the PCH-attached DMICs on this platform?

All DMICs are connected to the RT715. So is this patch even relevant?

Reply:
@plbossart Thanks for reminder. Do you mean we should not include 'platform/intel/dmic.m4'?